nerfstudio/data/dataparsers/nerfstudio_dataparser.py

```diff
@@ -458,6 +482,70 @@ def _load_3D_points(self, ply_file_path: Path, transform_matrix: torch.Tensor, s
         }
         return out
 
+    @staticmethod
+    def _align_points_to_target_plane(points: torch.Tensor, target_normal: torch.Tensor):
```

Comment:
Annotate the output type of this function.

Reply:
done